dev/CommonStyles/MediaTransportControls_themeresources.xaml

```diff
@@ -72,519 +94,12 @@
                                 <Setter Property="Style" Value="{ThemeResource CaptionTextBlockStyle }" />
                                 <Setter Property="IsTextScaleFactorEnabled" Value="False" />
                             </Style>
-                            <!-- Style for Position slider used in Media Transport Controls -->
-                            <Style x:Key="MediaSliderStyle" TargetType="Slider" BasedOn="{StaticResource DefaultSliderStyle}">
```

Comment:
@karenbtlai can we remove the resources we already had? I think we're kinda stuck with them because somebody might be using them already :)

Reply:
`MediaSliderStyle` is a resource that exists in the OS, so removing will default to referencing the OS resource if someone is using it.